src/sentry/api/serializers/models/project.py

```diff
@@ -98,13 +97,15 @@ def get_access_by_project(
 ) -> MutableMapping[Project, MutableMapping[str, Any]]:
     request = env.request
 
-    project_teams = list(ProjectTeam.objects.filter(project__in=projects).select_related("team"))
-    project_team_map = defaultdict(list)
+    project_teams = ProjectTeam.objects.filter(project__in=projects).values_list(
```

Comment:
This function takes sequence of projects. Is it possible to reduce this into sequence of project ids as well?

Reply:
there's no performance gain in doing so as we're not deserializing anything extra at the moment here